Sources/Widgets/Widgets3D/LineWidget/state.js

```diff
@@ -0,0 +1,45 @@
+import vtkStateBuilder from 'vtk.js/Sources/Widgets/Core/StateBuilder';
+
+export default function generateState() {
+  return vtkStateBuilder
+    .createBuilder()
+    .addStateFromMixin({
+      labels: ['moveHandle'],
+      mixins: ['origin', 'color', 'scale1', 'visible'],
+      name: 'moveHandle',
+      initialValues: {
+        scale1: 50,
+        origin: [1, 1, 1],
+        visible: false,
+      },
+    })
+    .addDynamicMixinState({
```

Comment:
the point with dynamic handles is that it starts with 0 handles. A static state here would display the handles before the user sets their position, whereas a dynamic state pops the handle display when the user sets its position. This behavior was inspired by DistanceWidget. I can see a way around this problem, by changing the visible mixin state of a static handle when the user places its line. Should I implement it?

Reply:
I think it's the best practice to have a static handle (which is displayed as soon as the mouse is on the view). When user click, then it creates a new handle if needed (for the angle widget, when there is 3 handle, then we cannot adding a new handle). So that's ok for me.

Reply:
After discussion with Laurenn, we came to the following solutions:
 - 1 dynamic mix for both handle (problem: how to configure the shape)
 - or 2 static mixin